packages/api/src/events/user/user_created.ts

```diff
@@ -114,3 +115,18 @@ export class PublishNewUserEvent implements EntitySubscriberInterface<Profile> {
     )
   }
 }
+
+@EventSubscriber()
+export class AddPopularReadsToNewUser
+  implements EntitySubscriberInterface<Profile>
+{
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+  listenTo() {
+    return Profile
+  }
+
+  async afterInsert(event: InsertEvent<Profile>): Promise<void> {
+    const isIOSUser = event.entity.user.source === 'APPLE'
```

Comment:
Discussion in Discord about this, but we will need to somehow get this from request header information here instead of from the auth provider source.